source/fundamentals/connection/mongoclientsettings.txt

```diff
@@ -421,6 +371,43 @@ This example specifies the following driver behavior in a pool of
    :emphasize-lines: 3-5
    :dedent:
 
+.. _mcs-logger-settings:
+
+Logger Settings
+---------------
+
+Chain the `applyToLoggerSettings() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html#applyToLoggerSettings(com.mongodb.Block)>`__
+method to modify the logging behavior of the driver.
+
+The following table describes all the methods you can chain to your
+settings to modify the driver's logger behavior:
+
+.. list-table::
+   :widths: 40 60
+   :header-rows: 1
+
+   * - Method
+     - Description
+
+   * - ``maxDocumentLength()``
+     - Sets the maximum document length, in characters, of a single log
+       message. The default value is 1000 characters.
+
+Example
+~~~~~~~
+
+This example specifies the following logging driver behavior:
+
+- The maximum number of characters for a single log message is set to
+  ``5000`` characters.
```

Comment:
Suggestion: Since there is only one list item, condense to a sentence

```suggestion
This example specifies that the maximum number of characters for a single log message is ``5000`` characters.
```